.changelog/2367.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+(Consul Enterprise) Add support to provide inputs via helm for audit log related configuration
```

Comment:
This should be changed to be the same changelog as the original PR (#2265). So:

.changelog/2265.txt